api/src/Controller/IncludeResource.php

```diff
@@ -0,0 +1,81 @@
+<?php declare(strict_types=1);
+/*.
+    require_module 'standard';
+.*/
+
+namespace App\Controller;
+
+use Slim\Container;
+use Slim\Http\Request;
+use Slim\Http\Response;
+
+class IncludeResource
+{
+
+    /**
+     * @var string
+     */
+    private $class;
+
+    /**
+     * @var string
+     */
+    private $field;
+
+    /**
+     * @var string
+     */
+    private $parameter;
+
+
+    public function __construct(string $class, string $parameter, string $field)
```

Comment:
I might be missing it, but where is this constructor being used?

Reply:
The constructor is used in each of the resources when it wants to signal something is included for example `Event/BaseEvent.php` line 115.